certbot-ci/certbot_integration_tests/utils/acme_server.py

```diff
@@ -226,16 +236,19 @@ def _prepare_boulder_server(self):
 
         print('=> Finished boulder instance deployment.')
 
-    def _prepare_http_proxy(self):
+    def _prepare_http_proxy(self) -> None:
         """Configure and launch an HTTP proxy"""
         print('=> Configuring the HTTP proxy...')
+        http_port_map = self.acme_xdist['http_port']
```

Comment:
On the hierarchy of methods to appease `mypy`, do you think `cast` is preferable to `ignore`? We could do something like:

```suggestion
        http_port_map = cast(Dict[str, int], self.acme_xdist['http_port'])
```
and avoid the `ignore[attr-defined]`.

Reply:
Yes I think `cast` is always better, because even if we "force" the type, at least we make an educated guess instead of just ignoring the problem. Thanks.